index.html

```diff
@@ -17,11 +17,11 @@
                     <a href="/">Welcome</a>
                 </div>
                 <div class="nav__links">
-                    <div class="nav__links-items">
-                        <a href="#">Portfolio</a>
-                        <a href="#">About</a>
-                        <a href="# ">Contact</a>
-                    </div>
+                    <ul class="nav__links-items">
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">About</a></li>
+                        <li><a href="# ">Contact</a></li>
+                    </ul>
                     <div class="nav__hamburger">
                         <img src="./images/Enabled@2x.svg" alt="hamburger menu">
                     </div>
```

Comment:
- [ ] Good job with the design. Kindly update the `font-size` and `font-weight` of the `h1` `Hey there. I'm...` to match the Figma template. 🤝️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/213629635-78a774bf-9390-48fd-9a14-1ce3567b04bf.png" width="400">|<img src="https://user-images.githubusercontent.com/17243325/213629628-422444f7-79c5-4b86-ac3d-3fddd53ddce4.png" width="400">|